go/vt/dbconfigs/dbconfigs.go

```diff
@@ -135,69 +135,102 @@ func registerPerUserFlags(dbc *userConfig, userKey string) {
 
 }
 
+// ConnParams contains mysql.connparams for now
```

Comment:
Let's also define a `Connect` function that gets the params and invokes mysql.Connect. There are lot of boilerplates in the code that call these in sequence.